src/style.css

```
@@ -1 +1,69 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars--0 {
+  display: flex;
+}
+
+.stars--1 {
+  display: flex;
+}
+
+.stars--2 {
+  display: flex;
+}
+
+.stars--3 {
+  display: flex;
+}
+
+.stars--4 {
+  display: flex;
+}
+
+.stars--5 {
+  display: flex;
+}
```

```suggestion
.stars {
  display: flex;
}
```
it will be style for .stars block